public/documents/upload_facilities.csv

```diff
@@ -1 +1 @@
-organization,facility_group,facility_name,facility_type,street_address (optional),village_or_colony (optional),zone_or_block (optional),district,state,country,pin (optional),latitude (optional),longitude (optional)
+organization,facility_group,facility_name,facility_type,street_address (optional),village_or_colony (optional),zone_or_block (optional),district,state,country,pin (optional),latitude (optional),longitude (optional),size (optional),enable_diabetes_management (true/false),enable_teleconsultation (true/false),teleconsultation_phone_number,teleconsultation_isd_code

```

Comment:
Reckon we should confirm the ordering of these new fields with Deepa?

Reply:
Not sure where to leave this comment 😅 Why don't we default the "enable" columns to false if blank? That way, as a user, I can stick "true" in the 5 facilities I want DM in, without having to fill in "false" in all the other rows.

Reply:
I thought about this but this will make the import slightly more verbose. These fields will need to be dealt separately before the import since our model doesn't allow nils. If setting all cells is something we can live with, we should stick to that imho and keep it simple.

Reply:
We could provide defaulting quite elegantly by inserting ourselves [here](https://github.com/simpledotorg/simple-server/blob/master/app/jobs/import_facilities_job.rb#L12) with something like:

```
import_facility = Facility.new(facility_params)

def facility_params
  facility.merge(
    enable_diabetes: facility[:enable_diabetes] || false
end
```

or even at the CSV parsing level [here](https://github.com/simpledotorg/simple-server/blob/master/app/models/facility.rb#L98-L112).

Speaking of CSV parsing, I think we need to do some work in this method to ensure we're parsing booleans correctly.

Reply:
`parse_facilities` is definitely **the** place to put this in. Rest addressed in comment below.

Reply:
Addressed in 0f9d452. ([Slack ref](https://simpledotorg.slack.com/archives/CFW5F0GFQ/p1590069415021900?thread_ts=1589901630.002500&cid=CFW5F0GFQ))